challenge_3/python/mindm/src/majority.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import sys
+from collections import defaultdict
+
+
+def args_to_list(arg_string):
+    """ Parses argument-string to a list
+    """
+    # Strip whitespace -> strip brackets -> split to substrings ->
+    # -> strip whitespace
+    arg_list = [x.strip() for x in arg_string.strip().strip("[]").split(',')]

```

Comment:
I noticed that when piping arguments in for example from a file, there would be some extra whitespace around the arguments. The other .strip() happens after the string is split to elements so that the spaces between commas (if any) are removed.